Psychtoolbox/PsychColorimetric/SPDToCCT.m

```diff
@@ -0,0 +1,55 @@
+function CCT = SPDToCCT(SPD,S_SPD)
+% CCT = SPDToCCT(SPD,S_SPD)
+%
+% Calculated Correlated Colour Temperature (CCT) from a spectral power
+% distribution.
+%
+% Demo 1:
+%
+% S_SPD = [380,5,81];
+% SPD = GenerateBlackBody(6500,SToWls(S_SPD));
+% CCT = SPDToCCT(SPD,S_SPD);
+%
+% Demo 2:
+%
+% load B_cieday
+% SPD = GenerateCIEDay(6500,B_cieday);
+% CCT = SPDToCCT(SPD,S_cieday); 
+% % This comes out at 6501 rather than 6500 for unknown reasons.
+%
+% Demo 3:
+%
+% load spd_houser.mat
+% CCT = SPDToCCT(spd_houser,S_houser);
+

```

Comment:
Maybe add some "History" comment like in other files with your date and name?
Btw. we assume all code contributions are implicitly MIT licensed unless otherwise noted and a different license is accepted in rare cases.